core/new-gui/src/app/dashboard/component/feature-container/saved-workflow-section/ngbd-modal-workflow-executions/ngbd-modal-workflow-executions.component.scss

```diff
@@ -30,3 +30,7 @@
   cursor: pointer;
   color: #007bff;
 }
+
+::ng-deep .cdk-overlay-container {
```

Comment:
what is this for? and why is it needed? can you maybe add comments to describe?

Reply:
This is for refactoring default z-index of ng-zorro to display tooltip and dropdown on top of the table.